pkg/server/event_handler_test.go

```diff
@@ -189,6 +189,33 @@ isOK: true`),
 			wantStatusCode:     http.StatusInternalServerError,
 			wantRespBodySubstr: "failed to unmarshal payloadMetadata",
 		},
+		{
```

Comment:
The meaning of this test case is not that obvious.

Reply:
I don't have strong opinion on it. 
@sailorlqh  What do you think?

Reply:
I think it won't hurt to add this test case to make sure the `logger.Infof()` is called without causing any issue.

Reply:
It is not obvious that this test case is to test`logger.Infof()` is called without causing any issue.

Reply:
Maybe rename the test case to `success_with_github_commit_missing`.

Reply:
updated test's name as `success_with_github_commit_missing`.

Reply:
Add a comment to explain this test case is to test `logger.Infof() is called without causing any issue.`

Reply:
A comment is added